h2o-algos/src/main/java/hex/generic/Generic.java

```diff
@@ -29,8 +29,6 @@
  */
 public class Generic extends ModelBuilder<GenericModel, GenericModelParameters, GenericModelOutput> {
 
-    private static final Class[] SUPPORTED_MOJOS = new Class[]{GlmMojoModel.class, GlmMultinomialMojoModel.class,
```

Comment:
Is there something that is replacing this check?

IMHO not all mojos are supported, or are they?

Reply:
Right now, all MOJOs are supported, but not tested. Basic metrics are available for all the MOJOs.

Reply:
Resolved, waiting for tests.